tests/newsletters/test_newsletter_views.py

```diff
@@ -12,12 +13,15 @@
 
 @pytest.mark.django_db
 def test_send_organisation(admin, client, project, user_factory,
-                           follow_factory):
+                           follow_factory, email_address_factory):
     organisation = project.organisation
     user1 = user_factory(get_newsletters=True)
-    user2 = user_factory()
+    user2 = user_factory(get_newsletters=True)
     user_factory()
-
+    email_address_factory(user=user1, email=user1.email,
+                          primary=True, verified=True)
+    email_address_factory(user=user2, email=user2.email,
+                          primary=True, verified=True)
     follow_models.Follow.objects.all().delete()
     follow_factory(creator=user1, project=project)
     follow_factory(creator=user2, project=project, enabled=False)
```

Comment:
oh right, that's why it's still only one email sent out...
But I still think it's a good idea to test all options why the email is not sent.

Reply:
Haha, oh my, being awake when reviewing would certainly help. :)